src/interface/report/index.js

```diff
@@ -23,42 +24,50 @@ const Report = props => (
           refreshReport={refreshReport}
         >
           {fight => (
-            <PlayerSelection
+            <BossEventLoader
```

Comment:
This will currently hold back player selection and all the rest until this is done. This slows down the initial load when opening a direct link to a player analysis page. Are there any objections for adding this to the EventParser next to the parser, character profile and events loaders so that it happens async with those calls?

Reply:
I thought of adding it here so that the data is loaded once per fight selection so it shouldn't be called again when changing player. I could have that logic wrong though

Reply:
Most common use case is a player reviewing multiple fights. Could also cache the result in the redux store, but even then since it's cached server side I am not sure if there's any need.

Reply:
Ok cool, I'll migrate the code to the EventParser